src/m365/spfx/commands/project/project-upgrade/Utils.ts

```diff
@@ -29,4 +29,7 @@ export class Utils {
       (typeof project.packageJson !== 'undefined' &&
       typeof project.packageJson.dependencies['knockout'] !== 'undefined');
   }
+  public static handleUTF8BOMEncoding(s: string): any {
```

Comment:
Let's move it to the global Utils so that we can use it everywhere in the CLI

Reply:
Good one ! had not realised thank you.